content/applications/general/email_communication/faq.rst

```diff
@@ -1,174 +1,185 @@
 No Error
 ********
 
-Odoo is not always capable of providing information for the reason it failed. The different
-providers implement a personalized policy of the bounce emails and it is not always possible
-for Odoo to interpret it correctly.
+Odoo is not always capable of providing information for the reason it failed. The different email
+providers implement a personalized policy of the bounce emails and it is not always possible for
+Odoo to interpret it correctly.
 
-If you have this problem on a recurring basis with the same client or the same domain, please
-do not hesitate to contact `Odoo Support <https://www.odoo.com/help>`_ for help in finding a reason.
+If this a problem on a recurring basis with the same client or the same domain, do not hesitate to
+contact `Odoo Support <https://www.odoo.com/help>`_ for help in finding a reason.
 
-Note: in such case, one of the most common reasons is related to :ref:`SPF <email_communication/spf_compliant>`
-and/or :ref:`DKIM <email_communication/DKIM_compliant>` configuration.
+.. note::
+  In such case, one of the most common reasons is related to :ref:`SPF
+  <email_communication/spf_compliant>` and/or :ref:`DKIM <email_communication/DKIM_compliant>`
+  configuration.
 
 Why is my email sent late?
 **************************
 
-It may happen that you schedule an email campaign but it is not sent on time. We know that
-we use a delayed job to send emails that we consider as not urgent (Newsletters concept
+It may happen that an email campaign is scheduled but it is not sent on time. We know that
+Odoo uses a delayed job to send emails that are considered as not urgent (Newsletters concept
 such as mass mailing, marketing automation, events). The system utility **cron** can be used
-to schedule programs to run automatically at predetermined intervals. We use that policy in order
-to avoid cluttering the mail servers and prioritize the communication.
+to schedule programs to run automatically at predetermined intervals. Odoo uses that policy in order
+to avoid cluttering the mail servers and instead prioritizes the communication.
 
-The emails considered urgent (communication from one person to another one such as
-Sales Orders, Invoices, Purchase Orders, etc.) are sent directly.
+The emails considered urgent (communication from one person to another one such as Sales Orders,
+Invoices, Purchase Orders, etc.) are sent directly (immediately).
 
 .. image:: faq/email-scheduled-later.png
    :align: center
    :alt: Email scheduled to be sent later.
 
-By default, the Mass Mailing cron runs every 60 minutes. So, you should wait maximum an hour
-before the campaign is actually sent.
+By default, the Mass Mailing cron runs every 60 minutes. This can be changed to no less than 5
+minutes, but for the reasons mentioned above this isn't recommended. To access this setting go to:
+:menuselection:`Settings --> Technical --> Automation --> Scheduled Actions`. The action name is
+`Mail: Email Queue Manager`
```

Comment:
```suggestion
By default, the Mass Mailing cron runs every 60 minutes. This can be changed to no less than 5
minutes. But, for the reasons mentioned above, this isn't recommended. To access this setting go to:
:menuselection:`Settings --> Technical --> Automation --> Scheduled Actions`. 

Select the action name, `Mail: Email Queue Manager,` and proceed to make any necessary adjustments.
```
quick question - is the Mass Mailing cron worthy of capitalization? If so, no worries. But would an Italics-route be useful here? Not quite sure - totally your call. What do you think?

Reply:
I think it would work with italics. Thanks for pointing this out!